.github/workflows/cloud_build_failure_reporter.yml

```diff
@@ -145,7 +147,7 @@ jobs:
                   // no periodic checks found across all commits, report it
                   createOrCommentIssue(
                     'Missing periodic',
-                    `Periodic test has not run in the past 24hrs. Last checked from ${
+                    `Periodic test is not found. Last checked from ${
```

Comment:
RE above comment: how are we reporting on both nightly(periodic) and continuous (on merge) triggers?

Reply:
updated logic